spring-xd-dirt/src/main/resources/META-INF/spring-xd/internal/admin-server.xml

```diff
@@ -6,11 +6,19 @@
 
 	<import resource="deployers.xml" />
 
+	<import resource="../sparkstreaming/container-filter.xml" />
```

Comment:
I think we can remove this line and the container-filter.xml file now that the SparkStreamingContainerFilter bean is provided directly on the ContainerMatcher.

Alternatively, if we want it to be more extensible, we could have ContainerMatcher find any ContainerFilter beans from the ApplicationContext similar to how this was implemented for the initial version of this PR (although I think that can wait until post 1.1)